bugbug/models/component.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, lemmatization=False):
             bug_features.Whiteboard(),
             bug_features.Patches(),
             bug_features.Landings(),
+            bug_features.Product(),
+            bug_features.Component(),
```

Comment:
We should get the values for when the bug was filed, not the current values. I'm not sure if the rollback guarantees that.

Reply:
Added two new features, `InitialProduct` and `InitialComponent` that go through the history of the bug: a9b0ed2cbbcfd38b515483fe05dc44e5dfe8946d. 

Here are the metrics: [metrics.log](https://github.com/user-attachments/files/16102039/output.log)

Overall it performs better than the current model, however it is slightly worse than the metrics I shared earlier when using only the `Product` and `Component` features.